bundle/src/start.ts

```diff
@@ -1,10 +1,28 @@
 // process.env.MONGOMS_DEBUG = "true";
 import cluster from "cluster";
 import os from "os";
+import { red, bold, yellow, cyan } from "chalk";
 import { initStats } from "./stats";
 
 // TODO: add tcp socket event transmission
 const cores = 1 || Number(process.env.threads) || os.cpus().length;
+const commit = require('child_process').execSync('git rev-parse HEAD').toString().trim();
```

Comment:
this doesn't work on systems, that don't have git installed/or added to their path.
catch this error and have a fallback